### PR TITLE
Add Part::location with full URL to source location #74

### PR DESCRIPTION
**Pull request by ruspl-afed**
TocChapter should not extend Part since we don't want to reference it
Encode Part#id and use it as persistent ID
Table#title should be containment reference
Table#rows should be containment reference

Signed-off-by: Alexander Fedorov <alexander.fedorov@arsysop.ru>

### CODE REVIEW

**Comment by ruspl-afed** [comment]
image

File: bundles/ru.arsysop.loft.rgm.cxxdraft/src/ru/arsysop/loft/rgm/internal/cxxdraft/ParagraphStructure.java [hunk 1]
```diff
@@ -71,7 +71,7 @@ private void readPoints(Paragraph paragraph, Element node) {
 		node.elements().stream() //
 				.filter(new IsDiv()) //
 				.filter(new OfClass("para")) //$NON-NLS-1$
-				.forEach(e -> new AppendPoint(context).accept(paragraph, node));
+				.forEach(e -> new AppendPoint(context).accept(paragraph, e));
```
Comment (ruspl-afed) [inline bundles/ru.arsysop.loft.rgm.cxxdraft/src/ru/arsysop/loft/rgm/internal/cxxdraft/ParagraphStructure.java:74]: This was a killer of Points

Comment (zelenyhleb) [reply bundles/ru.arsysop.loft.rgm.cxxdraft/src/ru/arsysop/loft/rgm/internal/cxxdraft/ParagraphStructure.java:74]: Working in a road will never be the same quality as working at home=)

**Review by zelenyhleb — APPROVED** [review]
Thank you!

**Pull request merged**